packages/core/src/extensions/NonEditableBlocks/NonEditableBlockPlugin.ts

```diff
@@ -34,6 +34,30 @@ export const NonEditableBlockPlugin = () => {
           }
           // Checks if key press is Enter
           if (event.key === "Enter") {
+            // Kind of a hacky way to ensure that pressing Enter when a file
```

Comment:
Hmm, this seems a bit hacky. I'm not a proponent of merging this as it makes the codebase worse for a minor win. We should invest more time in finding a cleaner solution (if we want to prioritize this(